src/Configuration/Config.Abstractions/src/ConfigurationPath.cs

```diff
@@ -23,6 +23,7 @@ public static class ConfigurationPath
         /// <returns>The combined path.</returns>
         public static string Combine(params string[] pathSegments)
         {
+            Do you even code bro ?
```

Comment:
Wat?

Reply:
Need a build error to verify that it's working. I'll remove it before merging.

Reply:
Yes, please remove 😺

Reply:
```suggestion
```